### DISCUSSION TITLE
世界卫生组织成立 OSPO 办公室

### DISCUSSION THREAD

**Discussion by Nliver** [kaiyuanshe/ONES#34] [日常讨论]
The World Health Organization is launching its Open Source Programme Office

https://socialimpact.github.com/insights/world-health-organization-OSPO-launch/

**Comment by wangleo61** [comment]
That's great news can be improved the OSS in world wide:
More equitable access to open source employment
In some countries, UN agencies are one of the largest employers of technologists. Creating an OSPO in a UN agency may increase open source software (OSS) employment opportunities in low- and middle-income countries in **sub-Saharan Africa, Latin America, and South/east Asia.**

**Comment by will-ww** [comment]
Good News~

政府、非盈利机构、高校等部门也会慢慢部署 OSPO 这样的组织，也是个趋势，里面所提到的一个 PPT 也挺有用~

[WHO OSPO Design Materials.pdf](https://github.com/kaiyuanshe/ONES/files/8254028/WHO.OSPO.Design.Materials.pdf)

**Reply by wangleo61** [reply]
PPT不错哈，点赞哈

**Comment by will-ww** [comment]
然后我发现这篇文章中提到的”数字公共品“也是个很好的研究对象：https://digitalpublicgoods.net/standard/

![image](https://user-images.githubusercontent.com/15010826/158395722-81e9612a-791c-4d64-a672-ab636b727baa.png)

**Reply by wangleo61** [reply]
欧洲走在前面哈
